src/predicttriton.py

```diff
@@ -0,0 +1,118 @@
+# Copyright 2022 Google LLC
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     https://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Simple Flask prediction for a model.
+
+Simple Flask prediction for a model.
+"""
+import argparse
+import os
+import subprocess
+from typing import List, Tuple
+
+from absl import app as absl_app
+from absl import flags
+from absl import logging
+from absl.flags import argparse_flags
+from flask import Flask
+from flask import request
+import gcsfs
+from tritonprocessor import T5TritonProcessor 
+
+app = Flask(__name__)
+FLAGS = flags.FLAGS
+
+flags.DEFINE_string("model_path", "t5-base", "path to the FT converted model on GCS or local filesystem. For example '/all_models/t5-v1_1-base'")
+flags.DEFINE_integer("port", 5000, "port to expose this server on. Default is '5000'.")
+flags.DEFINE_integer("triton_port", 8000, "local triton server port to proxy requests to. Default is '8000'.")
+flags.DEFINE_string("triton_host", "localhost", "Optional. Separate host to route triton requests to. Default is 'localhost'.")
+flags.DEFINE_string("hf_model_path", "t5-base", "path to the source model on HuggingFace. For example 'google/t5-v1_1-base'.")
```

Comment:
Since you are only using this for tokenizer loading, you could download the model and tokenizer config from the original model already uploaded to GCS instead of having the prediction server download it again from huggingface. This can happen in the conversion step. This way the converted model model will always have the original model config so that you can use it along.

Reply:
This was easier than expected to do, and has the added bonus of implicitly adding support for models not using the t5-base dictionary!

Done.